### DISCUSSION TITLE
Input Stops Registering

### DISCUSSION THREAD

**Discussion by isaaciverson** [AppGround-io/bluetooth-keyboard-and-mouse-support#253] [Bugs & Problems]
**Control Device:**
Pixel 4 XL, Android 13

**Receiving Device:**
Windows 11 Laptop

I connected to my laptop today via the app on my phone, and the app has stopped working. It says that there's a connection to my laptop, but when I try to move the mouse or activate any other controls, nothing registers on the laptop.

The app was working fine last time I used it.

Here are the troubleshooting steps I've already tried:
- Turning off Bluetooth and back on again on both devices
- Restarting both devices
- Closing the app and reopening it
- Going to try uninstalling and reinstalling the app, will report back.

Any feedback is appreciated, thanks!

**Comment by isaaciverson** [comment]
Just tried uninstalling and reinstalling the app. Mouse still doesn't move, even after the app's suggested troubleshooting steps.

Could it be an Android 13 compatibility issue?

**Reply by AppGround-io** [reply]
Please try the remove the Bluetooth pairing on both devices and pair the devices again through the app.

**Reply by isaaciverson** [reply]
Just tried what you suggested... Unfortunately it's still not working.
